corehq/apps/mobile_auth/views.py

```diff
@@ -60,6 +62,7 @@ def get_payload(self):
 
 @login_or_digest_or_basic_or_apikey()
 @require_GET
+@count_by_response_code('commcare.auth_keys.fetch.count')
```

Comment:
we can, though we may have to get rid of the url tagging if we don't want to pay for custom metrics